tox.ini

```diff
@@ -33,6 +33,7 @@ setenv =
     mpldev: UV_INDEX = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
     devdeps, mpldev: UV_INDEX_STRATEGY = unsafe-best-match # match pip's behavior
     fitsio: ASTROPY_ALWAYS_TEST_FITSIO = true
+    oldestdeps: PYTEST_ADDOPTS = -Wdefault -ra --doctest-rst --strict-config --strict-markers
```

Comment:
Here, I guess the idea is not to error on warnings, unlike for the regular tests. Is that so that we do not have to worry about warnings produced in older packages? 

But I don't know why the other options would be necessary...

Maybe easiest to resolve all this by adding a comment; the `tox.ini` files are hard to understand, as one basically has to have a concept of both `tox` and `uv` in one's head, when one does not regular look at either. So, all help is appreciated!!

Reply:
(for the record I see nox as a worse version of tox 🙊)